mkdocs.yml

```diff
@@ -192,6 +192,7 @@ nav:
   - External Resources:
       - Ansible Collection User Guide: https://docs.ansible.com/ansible/latest/user_guide/collections_using.html
       - Ansible User Guide: https://docs.ansible.com/ansible/latest/user_guide/index.html
+      - Authoring eos_cli_config_gen Role Guide: docs/contribution/authoring_eos_cli_config_gen.md
```

Comment:
This should be in the `Contributing to AVD:` section

Reply:
Thanks, fixed!